decrypt_test.go

```diff
@@ -6,18 +6,33 @@ import (
 )
 
 func TestDecrypt(t *testing.T) {
-	fixture := UnmarshalTestFixture(EncryptedTestFixture)
-	p7, err := Parse(fixture.Input)
-	if err != nil {
-		t.Fatal(err)
+	tests := []struct {
+		name      string
+		algorithm string
+		hashFunc  string
+		fixture   string
+		expected  []byte
+	}{
+		{name: "rsa-pkcs-#1-v1.5/1", algorithm: "RSA PKCS #1 v1.5", fixture: EncryptedTestFixture, expected: []byte("This is a test")},
```

Comment:
I'd use multiple lines when declaring these structs for clarity.

Reply:
Fixed in b1fb0e54346f2f8b218ca002427fe0eff22564f9